lib/contacts_service.dart

```diff
@@ -46,6 +46,16 @@ class ContactsService {
     return contacts.map((m) => Contact.fromMap(m));
   }
 
+  /// Loads the avatar for the given contact and returns it. If the user does
+  /// not have an avatar, then `null` is returned in that slot. Only implemented
+  /// on Android.
+  static Future<Uint8List> getAvatar(
+      final Contact contact, {final bool photoHighRes = true}) =>
+      _channel.invokeMethod('getAvatar', <String, dynamic>{
+        'contact': Contact._toMap(contact),
+        'photoHighResolution': photoHighRes,
```

Comment:
This should check whether the platform is correct before calling since this is Android only API. On iOS this causes a crash.  Not sure whether it is a plugin responsibility to make sure this is only called when `TargetPlatform == android` or should there be a stub provided on the iOS side.